web/source/keyboards/kmwkeyboards.ts

```diff
@@ -1518,11 +1518,13 @@ namespace com.keyman.keyboards {
      * and a keyboard stub must be registered before the keyboard is loaded
      * for the keyboard to be usable.
      *
-     * @param       {Object}      Pstub     Keyboard stub object
+     * @param       {Object}      kStub     Keyboard stub object
      * @return      {?number}               1 if already registered, else null
      */
-    _registerStub(Pstub): number {
+    _registerStub(kStub): number {
       var Lk;
+      // Make a copy of the stubs to mutate and register
+      let Pstub = Object.assign({}, kStub);
```

Comment:
Can we use modern ES2018 notation in our source (it should down-compile to the same code as you have here):

```suggestion
    _registerStub(Pstub): number {
      var Lk;
      // Make a copy of the stubs to mutate and register
      Pstub = { ... Pstub };
```

And no need to make a new variable which we then don't use -- that's just adding more variables for potential confusion later :)